tensorflow_datasets/core/registered.py

```diff
@@ -106,6 +107,14 @@
     version=r"[0-9]+\.[0-9]+\.[0-9]+",
 ))
 
+_skip_reg = True
+
+@contextmanager
+def skip_registeration():
+  global _skip_reg
+  _skip_reg = False
+  yield
```

Comment:
Note, it's good practice to wrap yield inside `try`/'finaly' block,

This make sure that `_skip_reg = True` is executed even when an exception is raised inside the context manager